doc/guides/contributing/pull-requests.md

```diff
@@ -573,12 +573,17 @@ If you want to know more about the code review and the landing process, see the
 ### Appendix: subsystems
 
 * `lib/*.js` (`assert`, `buffer`, etc.)
+* `benchmark`
+* `deps`
 * `build`
 * `doc`
 * `lib / src`
 * `test`
 * `tools`
+* `typings`
 
+Here is the full list of
+[subsystems](https://github.com/nodejs/core-validate-commit/blob/main/lib/rules/subsystem.js).
```

Comment:
Consider using `[subsystems][]` here to fit it into a line, and adding that link below.

Reply:
> Consider using `[subsystems][]` here to fit it into a line, and adding that link below.

Thank you for your advice.